fedora-toolbox

```diff
@@ -262,6 +264,64 @@ enter()
 )
 
 
+run_rm()
+(
+    if [ $1 = 1 ]; then
+        if ! rm_output=$($prefix_sudo podman $2 $3 --force 2>&42); then
+            echo "$0: failed to remove $3"
+            exit 1
+        fi
+    else
+        if ! rm_output=$($prefix_sudo podman $2 $3 2>&42); then
+            echo "$0: failed to remove $3"
+            exit 1
+        fi
+    fi
+    if ! [[ $rm_output = "" ]]; then
```

Comment:
Nitpick: would be better to use `[ "$rm_output" != "" ]` instead for POSIX compatibility.

Reply:
Done